document-generator-prosecution/src/main/java/uk/gov/companieshouse/document/generator/prosecution/UltimatumTemplateValues.java

```diff
@@ -0,0 +1,63 @@
+package uk.gov.companieshouse.document.generator.prosecution;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Holds the variable field values to be filled in the template for an Ultimatum
+ * letter.
+ */
+public class UltimatumTemplateValues {
+    @JsonProperty("DefendantName")
+    private String defendantName;
+
+    @JsonProperty("template_registry_addr")
```

Comment:
The JSON properties have different cases

Reply:
I'll fix that in the next round of work, when I rework the template.